common/lib/xmodule/xmodule/modulestore/django.py

```diff
@@ -243,9 +244,38 @@ class ModuleI18nService(object):
     i18n service.
 
     """
-
-    def __getattr__(self, name):
-        return getattr(django.utils.translation, name)
```

Comment:
Ok @mattdrayer , I think we should keep `__getattr__` here. I am not sure but may be it is possible that some xBlocks trying to access some other attribute of `django.utils.translation` instead of `ugettext`